services/brig/src/Brig/API/Public/Swagger.hs

```diff
@@ -41,6 +42,8 @@ type SwaggerDocsAPIBase = SwaggerSchemaUI "swagger-ui" "swagger.json"
 
 type VersionedSwaggerDocsAPI = "api" :> Header VersionHeader VersionNumber :> SwaggerDocsAPIBase
 
+type RealVersionedSwaggerDocsAPI = "real-api" :> Header' '[Required, Servant.Strict] VersionHeader VersionNumber :> SwaggerDocsAPIBase
```

Comment:
i like the idea, but i'm wondering if we should then even expose the unreal-api? instead we could move the frozen swagger files to unit tests and make sure things don't change in frozen versions, but always generate them dynamically.

i guess storing the files is faster, but how busy are these end-points?

Reply:
also note that real-api will differ from api in old versions because missing lower version bounds.